public/assets/css/dissaprovePageStyle.css

```diff
@@ -0,0 +1,81 @@
+*{
+    color: white;
+    font-weight: bold;
+}
+
+.container{
+    align-items: center;
+    flex-direction: column;
+}
+
+.content{
+    width: 671px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 100px;
+}
+
+.content span, p{
+    margin-bottom: 33px;
+}
+
+.formWrapper {
+    width: 100%;
+}
+
+form {
+    display:flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+span {
+    color: #F9B354;
+    font-weight: bolder;
+}
+
+.submitButtons{
+    display: flex;
+    justify-content: space-between;
+    margin-top: 33px;
+    font-size: 15px;
+}
+
+input[type="submit"], input[type="reset"] {
+	color: inherit;
+	border: none;
+	padding: 0;
+	font: inherit;
+	cursor: pointer;
+	outline: inherit;
+    width: 122px;
+    height: 51px;
+    font-weight: bolder;
+}
+
+textarea {
+    padding: 8px;
+    border: none;
+    resize: none;
+    color: grey;
+    outline: none;
+}
+
+a:link {
+    color: #F9B354;
```

Comment:
Geef de a een class of zorg dat je alleen de a pakt die in je div staat. Nu pakt die ook de a in de header